### DISCUSSION TITLE
Editions Needs for Artists

### DISCUSSION THREAD

**Discussion by a2kdefi** [metaplex-foundation/metaplex-program-library#1016] [Ideas]
I wanted to provide some input on the capabilities artists are looking for around editions.

1) Artists want editions to be mutable by default and want the update authority of the printed editions to be the same as the master edition update authority. Mutability on 1/1 has created some of the most amazing experiences for collectors such as an image getting updated in real-time based on the bidders profile pictures. 

2) Artists want to be able to reduce the max token supply of a limited edition on chain if they don't sell-out. It is really hard for artists to gauge the right amount of supply on a limited edition, so enabling an artist to reduce (not increase) the max token supply will help ensure NFT data is shown correctly across marketplaces. 

3) Artists want to be able to put a cap on the max token supply for an open edition. For example, let's say the artist wants to have a 24-hour timer on an open edition and when the timer expires the current supply becomes the max token supply.

**Comment by CryptoPlato** [comment]
This is greatly needed. Thanks!

**Comment by projectnamedate** [comment]
Happy to support!

**Comment by Bonyode** [comment]
bingx.com/en/activity/ge…

🚀 $MPLX is here in BingX to Multiply Gains, Maximize Power, Level Up the Xperience! 💥
#TradeMPLX  in #BingX
